Sources/Plasma/NucleusLib/pnMessage/plClientMsg.cpp

```diff
@@ -55,7 +55,7 @@ void plClientMsg::AddRoomLoc(const plLocation& loc)
     if (loc.IsValid())
         fRoomLocs.emplace_back(loc);
     else
-        hsStatusMessage("Trying to load an invalid room, ignoring");
+        hsAssert(false, "Trying to load an invalid room");
```

Comment:
The challenge with this is that it only fires in debug builds.

Reply:
Well, we could make it a hard error everywhere. I can't think of a good reason why you'd *want* to silently ignore invalid page loads, but perhaps there are weird cases that I don't know about.

Or just leave the original log message in place, in addition to the assert. Though I feel like the log message on its own isn't a good tool for uncovering these issues, considering that it has appeared on every game start for *years* without anybody paying attention to it...